### DISCUSSION TITLE
key combinations ?

### DISCUSSION THREAD

**Discussion by itagagaki** [1Remote/1Remote#764] [Q&A] [answered]
What is a “Key Combination” in “Resources”?
What can I do when I check this box?

**Comment by VShawn** [comment]
Uh I dont get the point, What do you mean `Key Combination`? where I can find it?

**Reply by itagagaki** [reply]
Here.
<img width="530" alt="image" src="https://github.com/user-attachments/assets/4a1e2b34-8d53-492e-8ba2-d41669a611ec">

Incidentally, 'Drives plugged in latter' near it is probably a typo of 'later'.

**Reply by VShawn** [reply, marked as answer]
> Key Combinations

Same as:

![image](https://github.com/user-attachments/assets/8509dad1-772f-44e8-8d72-cc1fc58e6e5d)


`'Drives plugged in latter' near it is probably a typo of 'later'.`

you are right.

**Reply by itagagaki** [reply]
Ah, I see. Thank you.

**Reply by Arpi** [reply]
Hello, 
Is it possible to set "Apply Windows key combinations" to "Only when using the full screen"? 
How? 
Thanks.

**Reply by VShawn** [reply]
This usage is not supported. But if you modify the code yourself to enable the RDP shortcut key switch only in full screen mode, it should be possible.
